pkgs/by-name/co/colorgrind/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  perl
+}:
+stdenvNoCC.mkDerivation {
+  pname = "colorgrind";
+  version = "0-unstable-2016-07-05";
+
+  src = fetchFromGitHub {
+    owner = "renatocf";
+    repo = "colorgrind";
+    rev = "6b68367c9713075a40c6719cb24217e9437ffa74";
+    hash = "sha256-KVR2IeoAkaTXDjEndPvrKUzOf4Zkha20vcH4TTJqXfM=";
+  };
+
+  installPhase =
+    let
+      perl' = perl.withPackages (ps: [ ps.TermANSIColor ]);
+    in
+    ''
+      install -Dm755 -t $out/bin colorgrind
+      substituteInPlace $out/bin/colorgrind \
+        --replace-fail "#!/usr/bin/perl" "#!${lib.getExe perl'}"
+    '';
+
+  meta = {
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

meta.maintainers must be set for new packages.